ChangeLog.d/fix-tls12server-sent-sigalgs.txt

```diff
@@ -0,0 +1,6 @@
+Bugfix
+    * Fix a bug whereby the the list of signature algorithms sent as part of the
+      TLS 1.2 server certificate request would get corrupted, meaning the first
+      algorithm would not get sent and an entry consisting of two random bytes
+      would be sent instead. Found by Serban Bejan <bejan.serban@gmail.com> and
+      Dudek Sebastian <sebastian.dudek@apator.com>
```

Comment:
Let's not spell out emails for spam bots to pick up. We're acknowledging the person (sometimes via a nickname such as their GitHub username), not inviting to contact them.

Reply:
I was pondering this, but we used this pattern in the past, so I emulated.